ocs_ci/deployment/zones.py

```diff
@@ -27,6 +27,21 @@ def are_zone_labels_missing():
     return len(nodes["items"]) == 0
 
 
+def are_zone_labels_present():
```

Comment:
can we somehow utilize ocs_ci.ocs.node.get_node_az?

Reply:
Looking at the source code, I'm not sure what that function actually does. I don't find it useful in this case.

Here I'm asking for nodes of given label via selector directly via `node_h.get(selector=ZONE_LABEL)` and I don't think that makes sense to replace this single line with another function, which uses different ocs class I don't actually need here.

Reply:
I could rewrite this to fetch all nodes, and then iterate over each node, checking it's label. That said, I'm going to merge it as it is to speed up the process here. But I can't directly use because of `ocs_ci.ocs.resources.ocs.OCS`, which I try to avoid because of it's design.